packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -493,6 +494,12 @@ export default class Extension {
     return true;
   }
 
+  private clearClipboard (rawTimeout: unknown) {
+    const timeout = typeof rawTimeout === 'number' ? rawTimeout : parseInt(String(rawTimeout || 0));
```

Comment:
perhaps:
```
const timeout = parseInt(String(rawTimeout || 0));
```
would be good enough :sweat_smile:

Reply:
not sure if you plan on changing this?

Reply:
Definitely! I even had it implemented, not sure how it vanished 😅 Thanks for pointing out 👍